scripts/globus/setup_globus.sh

```diff
@@ -54,14 +78,22 @@ cat << EOF > mapping.json
 EOF
 
 #DOMAINS="--domain ornl.gov --domain clients.auth.globus.org --domain gmail.com"
-DOMAINS="--domain ornl.gov"
+# For the Globus client to create the guest collection need to allow client
+# from the domain
+DOMAINS="--domain ornl.gov --domain clients.auth.globus.org"
+
+echo "{" > path_restriction.json
+echo "  \"DATA_TYPE\": \"path_restrictions#1.0.0\"," >> path_restriction.json
+echo "  \"read_write\": [\"${DATAFED_GCS_COLLECTION_ROOT_PATH}\"]" >> path_restriction.json
+echo "}" >> path_restriction.json
```

Comment:
Is this debug information or useful information for log files?

Reply:
Useful for log files  I believe. Do you have a recommendation?